python/tvm/topi/image/grid_sample.py

```diff
@@ -200,13 +200,13 @@ def _bilinear_sample(n, c, h, w):
 
     def _nearest_sample(n, c, h, w):
         y, x = _compute_source_index(n, h, w)
-        y_new = te.round(y).astype("int32")
-        x_new = te.round(x).astype("int32")
+        y_new = te.nearbyint(y).astype("int32")
```

Comment:
Can you comment a little about this change? 

The difference is now behavior can change by setting the rounding mode. Is this needed?

Reply:
The `round` fails as for `nearest` in cpu as interpolation onnx uses [`std::nearbyintf`](https://github.com/microsoft/onnxruntime/blob/62cc927f05580a8875af1fc93363d4b6d34bc2c6/onnxruntime/core/providers/cpu/tensor/grid_sample.cc#L189-L192)